dev-db/etcdctl/etcdctl-3.5.0.ebuild

```diff
@@ -0,0 +1,607 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The etcd command line client, v3, compatible with v2"
+HOMEPAGE="https://github.com/etcd-io/etcd"
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64 arm64"
+IUSE=""
+
+S=${WORKDIR}/etcd-${PV}
+
+inherit coreos-go-utils go-module
+
+# generated by merging /go.sum and /etcdctl/go.sum
+EGO_SUM=(
```

Comment:
Following the `eclass` doc, we can omit the hash (https://devmanual.gentoo.org/eclass-reference/go-module.eclass/index.html)

> For inclusion in EGO_SUM, the h1: value and other future extensions SHOULD be omitted at this time. The EGO_SUM parser will accept them for ease of ebuild creation.


```
sed -i 's/ h1.*$/"/' dev-db/etcdctl/etcdctl-3.5.0.ebuild
```